ticket_upgrade_prediction/models/base.py

```diff
@@ -1,4 +1,27 @@
 from abc import ABC, abstractmethod
+from typing import Union
+
+import numpy as np
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.linear_model import LinearRegression, LogisticRegression
+from sklearn.neighbors import KNeighborsClassifier, KNeighborsRegressor
+from xgboost import XGBClassifier, XGBRegressor
+
+# !TODO add lightgbm
+SklearnClassifier = Union[
+    LogisticRegression,
+    KNeighborsClassifier,
+    RandomForestClassifier,
+    XGBClassifier,
+]
+SklearnRegressor = Union[
+    LinearRegression,
+    KNeighborsRegressor,
+    RandomForestRegressor,
+    XGBRegressor,
+]
+
+SklearnModel = Union[SklearnClassifier, SklearnRegressor]
```

Comment:
is this used anywhere? whats the purpose

Reply:
Im testing it's usage for typing

Reply:
It is used for typing purposed, in `neural_net_model.py`